src/contracts/interfaces/IRegistryCoordinator.sol

```diff
@@ -7,11 +7,11 @@ pragma solidity =0.8.12;
  */
 interface IRegistryCoordinator {
     // DATA STRUCTURES
-    enum Status
+    enum OperatorStatus
     {
-        // default is inactive
-        INACTIVE,
-        ACTIVE
+        // default is DEREGISTERED
+        DEREGISTERED,
+        REGISTERED
```

Comment:
do we perhaps want `NEVER_REGISTERED` as a different status + the default? Seems like there might be cases where the distinction matters?
If not, I think `NOT_REGISTERED` would be more proper than `DEREGISTERED`.

Reply:
I like this idea a lot

Reply:
looks like its not needed as of now, but i'd like to keep it. easy change to make later

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/3304f12016c0455644619133f670db950103375f